bindings/java/c/host.c

```diff
@@ -74,6 +74,62 @@ static bool account_exists_fn(struct evmc_host_context* context, const evmc_addr
     return jresult != 0;
 }
 
+static enum evmc_access_status access_account_fn(struct evmc_host_context* context,
+                                                 const evmc_address* address)
+{
+    const char java_method_name[] = "access_account";
+    const char java_method_signature[] = "(Lorg/ethereum/evmc/HostContext;[B)I";
+
+    assert(context != NULL);
+    JNIEnv* jenv = attach();
+
+    // get java class
+    jclass host_class = (*jenv)->FindClass(jenv, "org/ethereum/evmc/Host");
+    assert(host_class != NULL);
+
+    // get java method
+    jmethodID method =
+        (*jenv)->GetStaticMethodID(jenv, host_class, java_method_name, java_method_signature);
+    assert(method != NULL);
+
+    // set java method params
+    jbyteArray jaddress = CopyDataToJava(jenv, address, sizeof(struct evmc_address));
+
+    // call java method
+    jint jresult =
+        (*jenv)->CallStaticIntMethod(jenv, host_class, method, (jobject)context, jaddress);
+    return (enum evmc_access_status)jresult;
```

Comment:
If we are not making it a bool, at least verify we fit the enum:
```suggestion
    assert(jresult == 0 || jresult == 1);
    return (enum evmc_access_status)jresult;
```